RecoMuon/MuonIdentification/python/calomuons_cfi.py

```diff
@@ -12,7 +12,7 @@
     inputMuons = cms.InputTag("muons1stStep"),
     inputTracks = cms.InputTag("generalTracks"),
     minCaloCompatibility = cms.double(0.6),
-    minPt = cms.double(1.0)
+    minPt = cms.double(1E9)
```

Comment:
did you check if this does anything?
Looking at implementation of CaloMuonProducer (since 2009), most of the configuration parameters are actually bogus.